examples/npm/discord/commands/eval.js

```
@@ -0,0 +1,27 @@
+    .setDescription('evaluate javascript (owner only)')
+    .addStringOption(o => o.setName('code').setDescription('code to evaluate').setRequired(true)),
+  ownerOnly: true,
+  run: async i => {
+    const code = i.options.getString('code', true);
+
+    let buf = '';
+    const sink = { write(s) { buf += s; return true; } };
+    const captured = new console.Console({ stdout: sink, stderr: sink });
+
+    try {
+      await eval(`(async (console) => { ${code} })`)(captured);
```

Biome (2.4.14)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)